index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <script src="./index.js"></script>
+    <link rel="stylesheet" href="./master.css">
+    <title>Server Lab</title>
+</head>
+<body>
+    <div id="container">
+      <div class="male_card">
+        <img src="https://www.w3schools.com/howto/img_avatar.png" alt="">
```

Comment:
instead of hardcoding it in your html file you might want to add that into your function in the js file. That way you'd have a "picture" for each person (right now you have 3 people, but two photos).